packages/devtools_app/lib/src/screens/provider/instance_viewer/instance_providers.dart

```diff
@@ -354,17 +345,19 @@ final AutoDisposeFutureProviderFamily<InstanceDetails, InstancePath>
         instance,
         eval: eval,
         isAlive: isAlive,
-        instanceRefId: instanceRef.id,
+        instanceRefId: instanceRef.id!,
         setter: setter,
       );
 
       if (enumDetails != null) return enumDetails;
 
-      final classInstance = await eval.getClass(instance.classRef, isAlive);
+      final classInstance =
+          await eval.safeGetClass(instance.classRef!, isAlive);
```

Comment:
Hoist instance.classRef into local variable

Reply:
I'll remove the other `instance.classRef!` reference instead, since after this line we have a  `classInstance` directly